device-types/Cisco/C9800-40-K9.yaml

```diff
@@ -14,21 +15,24 @@ interfaces:
   - name: Management Port
```

Comment:
In the configuration, what is the actual ports named? Is it actually called `Management Port`?

Reply:
Sorry, don't have access to a config at the moment, I had based the name "Management Port" off another model of 9800 already in the library, will try and find out the real name of this interface.

Reply:
Please let me know when you have found this information